Info.lua

```diff
@@ -1096,6 +1132,12 @@ g_PluginInfo =
 			RecommendedGroups = "admins",
 		},
 
+		["core.gamerule"] =
+		{
+			Description = "Allows players to set and query game rule values",
+			RecommendedGroups = "admins",
+		},
+
```

Comment:
Would it make sense to have a separate permission for querying, and a separate one for setting?